README.md

```diff
@@ -13,95 +13,19 @@ Appium You.i Engine Driver supports Android versions 4.1+ (API level 16+)
 npm install appium-youiengine-driver
 ```
 
-## Usage
-Import You.i Engine Driver, set [desired capabilities](http://appium.io/slate/en/1.5/?javascript#appium-server-capabilities) and create a session:
-
-```
-import { YouiEngineDriver } from `appium-youiengine-driver`
-
-let defaultCaps = {
-  app: 'path/to/your.app',
-  automationName: 'YouiEngine',
-  platformName: 'iOS'
-  deviceName: 'iPhone 6'
-  youiEngineAppAddress: 'localhost'
-};
-
-let driver = new YouiEngineDriver();
-await driver.createSession(defaultCaps);
-```
-Run commands:
-```
-console.log(await driver.getPageSource());
-
-```
 ## Desired Capabilities
```

Comment:
We should probably list in here the minimum required capabilities from iOS, Android and XCUITest drivers